objax/io/checkpoint.py

```diff
@@ -56,11 +62,14 @@ def __init__(self, logdir: str, keep_ckpts: int, makedir: bool = True, verbose:
             makedir: if True then directory for checkpoints will be created,
                 otherwise it's expected that directory already exists.
             verbose: if True then print when data is restored from checkpoint.
+            only_first_host_writes: if True then checkpoints are saved only by the first host in multi-host setup,
```

Comment:
Do I understand correctly: the code does not work if the option is set to `False`?
If that's indeed correct, why even make this an option?